site/get-data.js

```diff
@@ -0,0 +1,30 @@
+const swagImages = [];
+const fileNames = [];
+const swagList = require('../data.json').map(swag => {
+    // Generate unique filename
+    // @todo: do we want to force jpeg as the image format?
```

Comment:
I forced jpeg in the initial code to standardize all images, maybe we could in the future create variants like webp to optimize their sizes even more but I'm not sure about keeping the original format if it's what you're asking about

Reply:
I wasn't, but I was also thinking about that 😄 

I was asking more along the lines of if the image comes from the source as a PNG, is there a good reason to convert it to a jpeg (with the implication file type was actually considered by the source)?